README.md

```diff
@@ -197,15 +197,15 @@ The following checkers are available for finding components in binary files:
 | gpgme |gstreamer |gupnp |haproxy |hdf5 |hostapd |hunspell |
 | icecast |icu |irssi |kbd |kerberos |kexectools |libarchive |
 | libbpg |libdb |libgcrypt |libical |libjpeg_turbo |liblas |libnss |
-| libsndfile |libsoup |libsrtp |libssh2 |libtiff |libvirt |libvncserver |
-| libxslt |lighttpd |logrotate |lua |mariadb |mdadm |memcached |
-| mtr |mysql |nano |ncurses |nessus |netpbm |nginx |
-| node |ntp |open_vm_tools |openafs |openjpeg |openldap |openssh |
-| openssl |openswan |openvpn |p7zip |pcsc_lite |pigz |png |
-| polarssl_fedora |poppler |postgresql |pspp |python |qt |radare2 |
-| rsyslog |samba |sane_backends |sqlite |strongswan |subversion |sudo |
-| syslogng |systemd |tcpdump |trousers |varnish |webkitgtk |wireshark |
-| wpa_supplicant |xerces |xml2 |zlib |zsh | | |
+| librsvg | libsndfile |libsoup |libsrtp |libssh2 |libtiff |libvirt |
```

Comment:
Just noticed that format_checker linting rule is failing. You can manually run the script to format the table again.

Reply:
Done! Thankyou